esmvaltool/recipes/recipe_ocean_bgc.yml

```diff
@@ -174,12 +174,12 @@ diagnostics:
     variables:
       thetao: # Temperature 3D
         preprocessor: prep_timeseries_global_volume_average
-        field: TO3M
+        field: TO3Y

```

Comment:
There is no annual 3D temperature data provided in CMIP5. I'm not sure why this change was made. It may not matter as the `field` term is being abandoned soon, right?

Reply:
I think I've changed this because the OBS data you provided me as reference were all annual averages (1 timestep). Anyway, this will be dropped soon, so it's is really just used for naming the OBS data. I will revert the change anyway.